packages/datetime/src/datePicker.tsx

```diff
@@ -313,9 +319,9 @@ export class DatePicker extends AbstractPureComponent2<IDatePickerProps, IDatePi
             shortcuts === true
                 ? true
                 : shortcuts.map(shortcut => ({
-                      ...shortcut,
-                      dateRange: [shortcut.date, undefined],
-                  }));
+                    ...shortcut,
```

Comment:
you've unnecessarily removed whitespace. please run `yarn lint-fix` inside `packages/datetime`.

Reply:
The linting issue is resolved